patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -1,14 +1,40 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/EntityRenderer.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/EntityRenderer.java
-@@ -208,6 +208,7 @@
+@@ -91,6 +91,7 @@
+     private final MapItemRenderer field_147709_v;
+     private int field_78529_t;
+     private Entity field_78528_u;
++    private AxisAlignedBB pointedEntityBox;
```

Comment:
What is this field, and why isn't it at the bottom of the list of fields? Please comment this.